pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -966,9 +966,15 @@ func GetOpenshiftBootstrapClusterRoleBindings() []rbac.ClusterRoleBinding {
 		newOriginClusterBinding(StatusCheckerRoleBindingName, StatusCheckerRoleName).
 			Groups(AuthenticatedGroup, UnauthenticatedGroup).
 			BindingOrDie(),
-		newOriginClusterBinding(NodeRoleBindingName, NodeRoleName).
-			Groups(NodesGroup).
-			BindingOrDie(),
+		// Preserve the empty node binding with no subjects for tightening reconciliation
```

Comment:
Seems like we should have a `GetDeadClusterRoleBindings`.

Reply:
I wouldn't want to chance someone calling this and missing calling that

Reply:
We already trust callers to use `GetBootstrapClusterRoles` which gives them `GetDeadClusterRoles`.  I would expect `GetBootstrapClusterRoleBindings` to have the same logic.

Reply:
ah, ok

Reply:
created GetDeadClusterRoleBindings